pkg/shell/superuser.jsx

```diff
@@ -180,6 +187,32 @@ const UnlockDialog = ({ proxy, host }) => {
             <Button variant="secondary" className='btn-cancel' onClick={cancel}>
                 {_("Close")}
             </Button>);
+    } else if (bridges) {
+        const apply = () => {
+            start(bridge);
+        };
```

Comment:
These 4 added lines are not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-17536-20220708-114053-02201755-fedora-36-devel//Coverage/lcov/github-pr.diff.gcov.html)